README.md

```diff
@@ -46,6 +46,29 @@ a system CA bundle is unavailable.
 [openssl-probe]: https://github.com/alexcrichton/openssl-probe
 [webpki-roots]: https://github.com/rustls/webpki-roots
 
+## Deployment Considerations
+
+When choosing to use `rustls-platform-verifier` or another trust store option, there are important differences to consider. These
+are primarily about root certificate availability:
+
+| Backend                          | Updates                         | Roots used                                                                                            |
+|----------------------------------|---------------------------------|-------------------------------------------------------------------------------------------------------|
+| OS/platform (non-Linux/BSD)      | Live pushes                     | System store, with correct (dis)trust decisions from every source available.                          |
```

Comment:
Should this `Backend` column be labelled `rustls-platform-verifier (non-Linux/BSD)` to match the other backend options that are phrased in terms of specific crate combinations? I think if you read the full README it's clear that `rustls-platform-verifier` is using the OS/platform but it might be clearer to have the crate name here.

Reply:
Separately I also wonder about the word "correct" for describing the (dis)trust decisions. The other crates are not necessarily "incorrect".

Would a word like "full" or phrasing like "most accurate" be better?

Reply:
Is it not incorrect to only apply half or none of a distrust decision the rest of the platform is following?

Reply:
I don't think it's fair to describe webpki-roots or platform-verifier as incorrect, no. I would still prefer different wording here. There's nuance to the discussion that I think falls away with the binary correct/incorrect.

Reply:
That's valid! I probably took to strong of an approach here, and would prefer to leave it in a state where people who aren't satisfied with the description can come as more specific questions to fit their needs instead of making a blanket assumption. I've changed the wording to "full" per your suggestion.